apps/src/templates/sectionProgress/standards/LessonStatusDialog.jsx

```diff
@@ -0,0 +1,91 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import BaseDialog from '../../BaseDialog';
+import DialogFooter from '../../teacherDashboard/DialogFooter';
+import Button from '../../Button';
+import MultiCheckboxSelector from '../../MultiCheckboxSelector';
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  },
+  lessonListItem: {
+    display: 'flex',
+    flexDirection: 'row'
+  }
+};
+
+class LessonStatusDialog extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    handleConfirm: PropTypes.func.isRequired
+  };
+
+  render() {
+    return (
+      <BaseDialog
+        isOpen={this.props.isOpen}
+        handleClose={this.props.handleConfirm}
+        useUpdatedStyles
+        style={styles.dialog}
+      >
+        <h2>{i18n.updateUnpluggedLessonProgress()}</h2>
+        <p>{i18n.updateUnpluggedLessonProgressSubHeading()}</p>
+        <h3>{i18n.completedUnpluggedLessons()}</h3>
+        <MultiCheckboxSelector
+          noHeader={true}
+          items={[
+            {
+              id: 'one',
+              name: 'Lesson 1',
+              url: 'https://curriculum.code.org/csf-19/coursea/1/'
```

Comment:
totally fine as a placeholder but want to double check,  will these links go to the curriculum.code.org lesson plan or to the actual level on code studio?

Reply:
I think they should go directly to the lesson plan since they are unplugged lessons they usually wont have as much information about the lesson in code studio levels